app/models/user.rb

```diff
@@ -29,8 +29,7 @@ def create_ldap_user
     # check to see if this is because the LDAP auth succeeded, or if we're coming from registration
     # we do this by performing an LDAP search for the new user. If it fails, we need to create the
     # user in LDAP
-    cfg_file = ::Devise.ldap_config || Rails.root.join("config", "ldap.yml")
-    ldap_config = YAML.safe_load(ERB.new(File.read(cfg_file)).result)[Rails.env]
+    ldap_config = Velum::LDAP.ldap_config
```

Comment:
We can certainly refactor this after the merge so we focus only on the functionality being introduced, but I think this would be good to be read on an initializer (e.g. `config/initializers/ldap.rb`) so we only load this once during the initialization phase of the application.